components/debug.rst

```diff
@@ -88,4 +88,34 @@ Using the ``DebugClassLoader`` is done by calling its static
 
     DebugClassLoader::enable();
 
+The ``DebugClassLoader`` also does lightweight static code analysis on the
```

Comment:
does seems not that needed

Reply:
What do you mean? I don't get it ^^

Reply:
Thank you for reviewing btw!

Reply:
the word "does" seems not that needed

Reply:
What other verb would you use? If I remove "does", the sentence has no verb ^^